src/hooks/useLogin.js

```diff
@@ -0,0 +1,35 @@
+import { GoogleAuthProvider, signInWithPopup } from 'firebase/auth';
+import { auth } from '../configs/firebase';
+import { useAuthContext } from './useAuthContext';
+import { LOGIN } from '../contexts/AuthContext';
+
+const googleProvider = new GoogleAuthProvider();
+
+export const useLogin = () => {
+  const { dispatch } = useAuthContext();
+
+  const login = async () => {
+    console.log('clicked');
+    try {
+      const result = await signInWithPopup(auth, googleProvider);
+      // const credential = GoogleAuthProvider.credentialFromResult(result);
+      // const token = credential.accessToken;
```

Comment:
Can we please remove this comments? If this variables are require ltr we can use create them at that time.

Reply:
It's still under process. Figuring out when and where to use them.